deploy.yaml

```diff
@@ -22,7 +21,7 @@
               -e PAGURE_USER_TOKEN={{ pagure_user_token }} \
               -e PAGURE_FORK_TOKEN={{ pagure_fork_token }} \
               -e PYTHONPATH=/src \
-              -v {{ playbook_dir }}:/src:Z \
+              -v {{ playbook_dir }}:/src \
```

Comment:
Without it I'm seeing:

```python
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/git/__init__.py", line 83, in <module>
    refresh()
  File "/usr/local/lib/python3.7/site-packages/git/__init__.py", line 73, in refresh
    if not Git.refresh(path=path):
  File "/usr/local/lib/python3.7/site-packages/git/cmd.py", line 290, in refresh
    raise ImportError(err)
ImportError: Bad git executable.
The git executable must be specified in one of the following ways:
    - be included in your $PATH
    - be set via $GIT_PYTHON_GIT_EXECUTABLE
    - explicitly set via git.refresh()

All git commands will error until this is rectified.

This initial warning can be silenced or aggravated in the future by setting the
$GIT_PYTHON_REFRESH environment variable. Use one of the following values:
    - quiet|q|silence|s|none|n|0: for no warning or exception
    - warn|w|warning|1: for a printed warning
    - error|e|raise|r|2: for a raised exception

Example:
    export GIT_PYTHON_REFRESH=quiet


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/packit", line 7, in <module>
    from packit.cli.packit_base import packit_base
  File "/usr/local/lib/python3.7/site-packages/packit/cli/packit_base.py", line 6, in <module>
    from packit.cli.sourcegit_to_dist_git import sg2dg
  File "/usr/local/lib/python3.7/site-packages/packit/cli/sourcegit_to_dist_git.py", line 5, in <module>
    from packit.config import pass_config, get_context_settings, get_local_package_config
  File "/usr/local/lib/python3.7/site-packages/packit/config.py", line 17, in <module>
    from packit.utils import exclude_from_dict
  File "/usr/local/lib/python3.7/site-packages/packit/utils.py", line 8, in <module>
    import git
  File "/usr/local/lib/python3.7/site-packages/git/__init__.py", line 85, in <module>
    raise ImportError('Failed to initialize: {0}'.format(exc))
ImportError: Failed to initialize: Bad git executable.
```

Reply:
the problem is that you can't do :Z with rootless:
```
$ podman run --rm -ti -v $PWD:/src:Z registry.fedoraproject.org/fedora:29 bash
relabel failed "/home/tt": operation not permitted

$ podman run --rm -ti -v $PWD:/src registry.fedoraproject.org/fedora:29 bash
[root@8150111558ec /]# echo hello!
hello!
```

Reply:
Really?

```bash
$ podman run --rm -ti -v $PWD:/src:Z registry.fedoraproject.org/fedora:29 bash
[root@9994a6caaf41 /]# echo x
x
$ getenforce 
Enforcing
```

Reply:
I'd say it's worth reporting upstream the relabel failure, I'll correct the other Z's.

Reply:
> reporting upstream the relabel failure

I don't understand. I don't see any failure worth reporting. OK, you can't (I don't know why) relabel whole HOME, but we don't need that, we don't volume mount whole HOME, but just the packit directory which CAN be mounted and thus relabeled. 

> I'll correct the other Z's.

Please don't, nothing works without it.

Reply:
This is the problem why it does not work for me:
```bash
packit$ ls -Z -d .
system_u:object_r:container_file_t:s0:c111,c632 .
```

Reply:
Mine says:

```
packit$ ls -Z -d .
system_u:object_r:container_file_t:s0:c0,c1007 .
```